docs/release.md

```diff
@@ -27,13 +27,13 @@ It is possible that a core approver isn't a contrib approver. In that case, the
 
 1. Update Contrib to use the latest in development version of Core. Run `make update-otel` in Contrib root directory and if it results in any changes submit a draft PR to Contrib. Ensure the CI passes before proceeding. This is to ensure that the latest core does not break contrib in any way. We’ll update it once more to the final release number later.
 
-2. Determine the version number that will be assigned to the release. Usually, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.85.0` as the version to be released, following `v0.84.0`.
+2. Determine the version number that wil be assigned to the release. Usually, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.85.0` as the version to be released, following `v0.84.0`.
```

Comment:
This adds a typo?

Reply:
omg... thanks i added it to test spellcheck lol and left it there 🤦🏻

Reply:
@atoulme fixed.... although you know fixing 3 typos adding 1, technically still making things better 😆

Reply:
Good running average